helmfile/values/podsecuritypolicies/common/rook-ceph.yaml.gotmpl

```diff
@@ -0,0 +1,234 @@
+          - emptyDir
+          - secret
+          - projected
+          - hostPath
+        allowedHostPaths:
+          - pathPrefix: "/var/lib/rook"
+        runAsUser:
+          rule: RunAsAny
+        runAsGroup:
+          rule: RunAsAny
+        supplementalGroups:
+          rule: RunAsAny
+        fsGroup:
+          rule: RunAsAny
+        allowedCapabilities:
+          - SYS_ADMIN
+    rook-ceph-mon:
+      podSelectorLabels:
+        app: rook-ceph-mon
+      allow:
+        allowPrivilegeEscalation: true
+        volumes:
+          - secret
+          - projected
+          - hostPath
+        allowedHostPaths:
+          - pathPrefix: "/var/lib/rook"
+        runAsUser:
+          rule: RunAsAny
+        runAsGroup:
+          rule: RunAsAny
+        fsGroup:
+          rule: RunAsAny
+        supplementalGroups:
+          rule: RunAsAny
+        allowedCapabilities:
+          - SYS_ADMIN
+    rook-ceph-tools:
+      podSelectorLabels:
+        app: rook-ceph-tools
+      allow:
+        volumes:
+          - configMap
+          - emptyDir
+          - projected
+        runAsUser:
+          rule: MustRunAsNonRoot
+      mutation:
+        fsGroup: 2016
+    rook-ceph-detect-version:
+      podSelectorLabels:
+        app: rook-ceph-detect-version
+      allow:
+        privileged: true
+        allowPrivilegeEscalation: true
+        volumes:
+          - emptyDir
+          - projected
+        runAsUser:
+          rule: RunAsAny
+        runAsGroup:
+          rule: RunAsAny
+        supplementalGroups:
+          rule: RunAsAny
+        fsGroup:
+          rule: RunAsAny
+        allowedCapabilities:
+          - SYS_ADMIN
+    rook-ceph-csi-detect-version:
+      podSelectorLabels:
+        app: rook-ceph-csi-detect-version
+      allow:
+        privileged: true
+        allowPrivilegeEscalation: true
+        volumes:
+          - emptyDir
+          - projected
+        runAsUser:
+          rule: RunAsAny
+        runAsGroup:
+          rule: RunAsAny
+        supplementalGroups:
+          rule: RunAsAny
+        fsGroup:
+          rule: RunAsAny
+        allowedCapabilities:
+          - SYS_ADMIN
+    rook-ceph-osd-prepare:
+      podSelectorLabels:
+        app: rook-ceph-osd-prepare
+      allow:
+        privileged: true
+        allowPrivilegeEscalation: true
+        volumes:
+          - emptyDir
+          - projected
+          - hostPath
+        allowedHostPaths:
+          - pathPrefix: "/"
+        runAsUser:
+          rule: RunAsAny
+        runAsGroup:
+          rule: RunAsAny
+        supplementalGroups:
+          rule: RunAsAny
+        fsGroup:
+          rule: RunAsAny
+        allowedCapabilities:
+          - SYS_ADMIN
+mutationExceptions:
+  - app: csi-rbdplugin
+  - app: csi-rbdplugin-provisioner
+  - app: rook-ceph-mgr
+  - app: rook-ceph-mon
+  - app: rook-ceph-osd
+  - app: rook-ceph-detect-version
+  - app: rook-ceph-csi-detect-version
+  - app: rook-ceph-osd-prepare
```

Comment:
What mutation was causing the issue?

Reply:
Wonder if it is adding a override for that mutation, so not all mutations are overridden.

Reply:
I did not find much success when trying to override mutations for these pods, they would either be stuck crashing, or for at least some of the pods there were issues with supplemental group in terms of violating constraints.